### PR TITLE
Feature/improved map editor

### PR DESCRIPTION
**Pull request by cadw**
Removed unneccessary informaton and made the interface look a little bit nicer. :)

### CODE REVIEW

**Comment by hungnguyen94** [comment]
The table row has a lot of unnecessary height. This happened after adding the delete button. Also, the table still has inline css styling. 

Is it maybe better to change the table to a list? 
I initially used a table to display all the data of the camera and player entities. But since they're now hidden in a collapse, it could as well be a list.

File: src/main/webapp/app/map/map-table-view-template.html [hunk 1]
```diff
@@ -7,14 +7,14 @@
...
-        <th colspan="3"><b>Cameras</b></th>
+        <th colspan="3"><h3>Cameras</h3></th>
     </tr>
     </thead>
     <tbody>
     <tr ng-repeat-start="camera in vm.map.cameras"
         ng-click="vm.setSelected(camera)"
         ng-class="{ 'success': vm.selected == camera }">
-        <td colspan="2">{{camera.name}}</td>
+        <td colspan="2"><h4 class="text-muted"><i class="glyphicon glyphicon-facetime-video"></i> &nbsp; {{camera.name}}</h4></td>
```
Comment (ChrisBerg) [inline src/main/webapp/app/map/map-table-view-template.html]: &nbsp seems a bit ugly, couldn't this be done in the CSS?

File: src/main/webapp/app/map/load-background-dialog.html [hunk 2]
```diff
@@ -3,7 +3,7 @@
     <div class="modal-header">
         <button type="button" class="close" data-dismiss="modal" aria-hidden="true"
                 ng-click="vm.clear()">&times;</button>
-        <h4 class="modal-title" id="myMapLabel">Load an image as background</h4>
+        <h4 class="modal-title" id="myMapLabel"> Load an image as background</h4>
```
Comment (rubenwiersma) [inline src/main/webapp/app/map/load-background-dialog.html]: Space would be better outside of the h4 tag (I think)

File: src/main/webapp/app/map/map-table-view-template.html [hunk 3]
```diff
@@ -76,9 +72,6 @@
         <!--<tr ng-repeat-end uib-collapse="false">-->
```
Comment (rubenwiersma) [inline src/main/webapp/app/map/map-table-view-template.html]: Commented code

File: src/main/webapp/app/map/map-table-view-template.html [hunk 4]
```diff
@@ -51,15 +47,15 @@
...
     <tr>
-        <th colspan="3"><b>Players</b></th>
+        <th colspan="3"><h3>Players</h3></th>
     </tr>
     </thead>
     <tbody>
     <tr ng-repeat-start="player in vm.map.players"
         ng-click="vm.setSelected(player)"
         ng-class="{ 'success': vm.selected == player }">
         <!--<td>{{player.id}}</td>-->
```
Comment (rubenwiersma) [inline src/main/webapp/app/map/map-table-view-template.html]: Commented code

File: src/main/webapp/app/map/map-table-view-template.html (hunk 1 above)
Comment (cadw) [inline src/main/webapp/app/map/map-table-view-template.html]: There is no direct alternative for &nbsp;, however we could use margin-left.

File: src/main/webapp/app/map/load-background-dialog.html (hunk 2 above)
Comment (cadw) [inline src/main/webapp/app/map/load-background-dialog.html]: As discussed with you today, we won't change this.

File: src/main/webapp/app/map/map-table-view-template.html (hunk 1 above)
Comment (cadw) [inline src/main/webapp/app/map/map-table-view-template.html]: Advantage of nothing but space is that it looks the same in all browsers, so I think it is better to just leave it the way it is.

**Pull request merged**